jbpm-workitems/jbpm-workitems-webservice/src/main/java/org/jbpm/process/workitem/webservice/WebServiceWorkItemHandler.java

```diff
@@ -363,6 +363,8 @@ public void executeWorkItem(WorkItem workItem,
             if (client == null) {
                 throw new IllegalStateException("Unable to create client for web service " + interfaceRef + " - " + operationRef);
             }
+            
+            addCDataWriterInterceptor(workItem, client);
```

Comment:
I think this call is not needed, you are already adding the interceptor when the client is created in createClient.

Reply:
right